src/project/internal/notationproject.cpp

```diff
@@ -566,16 +567,17 @@ mu::Ret NotationProject::doSave(const io::path_t& path, engraving::MscIoMode ioM
 
     // Step 1: check writable
     {
-        if (fileSystem()->exists(savePath) && !fileSystem()->isWritable(savePath)) {
+        if ((fileSystem()->exists(savePath) && !fileSystem()->isWritable(savePath))
+            || (fileSystem()->exists(targetContainerPath) && !fileSystem()->isWritable(targetContainerPath))) {
```

Comment:
As mentioned earlied, this can get shortened to
```cpp
        if ( !fileSystem()->isWritable(savePath) || !fileSystem()->isWritable(targetContainerPath)) {
```
IMHO there's no need to test for existence, as isWriteable() does that too

Reply:
The existence test was there before and I'm not totally comfortable removing it without fully understanding its consequences. Besides, I'm not sure if testing if a file is writable when it exists is equivalent to testing only if it is writable (existing or not).